### PR TITLE
Update CTAs

### PR DESCRIPTION
**Pull request by caldav**
Adjust wording  and bring clarity to the multiple CTAs by turning them into a list of steps, walking the user through:
  1. Installing snapd (which brings usage guidance as well)
  2. Going through the first snap tutorial (which then branches out to plugins and general snapcraft docs)
  3. Linking GitHub repos to bring visibility to the build beta

## Screenshots

screenshot from 2017-04-18 12-42-25

### CODE REVIEW

**Comment by caldav** [comment]
@evandandrea feedback appreciated

**Comment by evandandrea** [comment]
Definitely an improvement. +1.

**Comment by caldav** [comment]
Holding on for build.s.io

File: index.html [hunk 1]
```diff
@@ -8,11 +8,13 @@
...
                 server, cloud or device, and deliver updates directly.</p>
-                <p><a href="/create">Learn to craft snaps</a>
-                or <a href="/community">join the community</a></p>
-                <div class="clearfix">
-                    <p>Read the <a href="/docs/core/install">installation instructions</a> or <a href="http://www.github.com/snapcore/snapd">build <code>snapd</code> from source</a></p>
-                </div>
+                <p>
+                  <ol class="list">
+                    <li>1. Read the <a href="/docs/core/install">installation instructions</a></li>
+                    <li>2. Follow the tutorial to <a href="https://tutorials.ubuntu.com/tutorial/create-first-snap#0">create your first snap</a></li>
```
Comment (anthonydillon) [inline index.html]: Add `class="external"` to the tutorials link please.

File: index.html [hunk 2]
```diff
@@ -8,11 +8,13 @@
...
+                    <li>3. <a href="https://build.snapcraft.io">Link your GitHub repos</a> to automate builds and releases</li>
```
Comment (anthonydillon) [inline index.html]: Also needs an external class.

**Comment by caldav** [comment]
@anthonydillon indeed! Done.

**Comment by pmahnke** [comment]
@caldav I think this looks pretty strange... and the current site too.

We should strive for 1, max 2 CTAs really.

Let's use the three blocks below

image

**Comment by evandandrea** [comment]
@pmahnke could we instead have two CTAs? Something like:

- Learn to craft your first snap
- Auto-build and publish from your GitHub repos

**Comment by pmahnke** [comment]
@evandandrea that's sounds good, but let's keep the three links below as well.

**Comment by caldav** [comment]
WFM, changing

**Comment by caldav** [comment]
screenshot from 2017-04-19 14-50-59

Thoughts?

**Comment by evandandrea** [comment]
👍

**Pull request merged**